gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CompileRefasterTask.java

```diff
@@ -38,12 +38,15 @@ public CompileRefasterTask() {
         // Don't care about .class files
         setDestinationDir(getTemporaryDir());
 
-        // Ensure we hit the non-incremental code-path since we override it
-        getOptions().setIncremental(false);
+        // Ensure we hit the incremental code-path since we override it
+        getOptions().setIncremental(true);
     }
 
     @Override
-    protected final void compile(InputChanges inputs) {
+    // TODO(forozco): override compile(InputChanges inputs) once we can raise our minimum version 6.0
+    @SuppressWarnings("deprecated")
+    protected final void compile(IncrementalTaskInputs inputs) {
+        getProject().getLogger().warn("Doing compilation - Felipe");
```

Comment:
DOING COMPILATION FELIPE

Reply:
🤦‍♂

Reply:
cancelled publish to avoid people accidentally getting spammed. I'll try to be better going forward

Reply:
... I don't know how I missed this 🤦‍♂

Reply:
https://github.com/palantir/gradle-baseline/pull/908